pages/datasets/[dataset].js

```diff
@@ -0,0 +1,551 @@
+/* eslint-disable guard-for-in, no-restricted-syntax, no-use-before-define, eqeqeq */
+
+import fetch from 'isomorphic-unfetch';
+import Head from 'next/head';
+import Papa from 'papaparse';
+import React from 'react';
+import ReactTooltip from 'react-tooltip';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+import BarChart from '../../components/charts/chartsjs/BarChart';
+import ChartNote from '../../components/charts/chartsjs/ChartNote';
+import DoughnutChart from '../../components/charts/chartsjs/DoughnutChart';
+import DatasetDetails from '../../components/explore-the-data-page/DatasetDetails';
+import FilterPanel from '../../components/explore-the-data-page/FilterPanel';
+import HeroContent from '../../components/explore-the-data-page/HeroContent';
+import Layout from '../../components/Layout';
+import datasets from '../../data/datasets';
+import theme from '../../theme';
+
+export default class Explore extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      activeDataset: '',
+      data: {},
+      filters: {},
+    };
+
+    this.updateFilters = this.updateFilters.bind(this);
+    this.updateFilterGroup = this.updateFilterGroup.bind(this);
+  }
+
+  /**
+   * Once component mounts, check for query string and load
```

Comment:
Should we update this comment now that we're not using query strings?

Reply:
Updated and also removed the no longer necessary `componentDidUpdate` lifecycle.